helm/values.yaml

```diff
@@ -0,0 +1,70 @@
+config:
+  hostname: 192.168.49.2
```

Comment:
Hostname ist bei einem Cluster missverständlich. Sollten wir vielleicht später überarbeiten.
Eine spezielle IP für den Hostname führt wahrscheinlich auch eher in die Irre. Könnte 'localhost' für alle Nutzer ein besserer Beispiel-Hostname sein?